openedx_rocketchat/views.py

```diff
@@ -0,0 +1,88 @@
+import hashlib
+import logging
+
+from django.http import Http404
+from django.contrib.auth.decorators import login_required
+
+from openedx_rocketchat.edxapp_wrapper.edxmako_module import render_to_response
+from openedx_rocketchat.edxapp_wrapper.get_configuration_helpers import get_configuration_helpers
+from openedx_rocketchat.edxapp_wrapper.get_courseware import get_course_with_access, has_access
+from openedx_rocketchat.edxapp_wrapper.get_opaque_keys import get_course_from_string
+from openedx_rocketchat.edxapp_wrapper.get_student_models import get_course_enrollment, get_course_enrollment_manager
+
+from .utils import (
+    create_course_group,
+    initialize_api_rocket_chat,
+    get_rocket_chat_settings,
+    create_token,
+)
+
+LOG = logging.getLogger(__name__)
+ROCKET_CHAT_DATA = "rocket_chat_data"
```

Comment:
Since, this it is a constant used here and openedx_rocketchat/utils.py file, is better to have a global settings, agree? @andrey-canon

Reply:
yes, I do